### PR TITLE
Add ruleset for persisting charges

### PR DESCRIPTION
**Pull request by prtandrup**
## Description

This PR does the following: 
- Adds ruleset process flow for persisting charges to SQL database (both .png and .drawio files)
- Adds ruleset process flow and description to the process-flows folder's README.

Please note: 
- the datamodel diagram will be added in a later PR, and with this, a reference will be made from the README mentioned above to the datamodel diagram.

## References

* #207

### CODE REVIEW

**Review by bemwen — APPROVED** [review]

File: docs/process-flows/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ All process flows within the Charges domain will be gathered here.
 | Process flows |
 |-------------------|
 |[Change of charge price list](#Change-of-charge-price-list-flow)|
+|[Persisting charges rule set](#Persisting-charges-rule-set)|
```
Comment (BjarkeMeier) [inline docs/process-flows/README.md:8]: btw: I don't think that there is a path from /README.md to docs/process-flows, so the only way to find this doc for now if is people are poking around. Perhaps we could at least add a note about looking inside the /docs folder?

File: docs/images/PersistingChargesRuleSet_ProcessFlow.drawio [hunk 2]
```diff
@@ -0,0 +1 @@
...jQuoIsjJC+auyW6YOwDEmmaLqhlwuhZ9JLc80sJmJWiaZoKPkG4y9sFyWOTPvm629DETVsbaSzWIx8KWXG9f1yMaxZPcVVwUlxdgA5tu2hulWMOi8dL5jJ+l1POJSqM1yVRdUe9qqhYpjLZntl5fE5L6CLXLwFbMGC49gs40ATBvwRq22DH6ETdyGwRKFTQpJd3H32Zn6hpR53JdVy/h0IVjLAVWzE7dxcl41MjJR9Fx16NgZTPCkrr3KTTmgKqzd0RQ5H3FfJDDsnTHfYc89y5V9K2hDErCG7C6bvTfh19433xhSqKOcmXvObgJSddO+7rBDRywIUf/auQdKrCEXBkoi17m1QEnjA6XHIMIPxPbdBHCRPmpEXr6hKWhUElRVW5WSzFmsUZYqhmcnYVUvv1NFVUQvNLAFIrbaErF+iY5OI4MuzO4JEui1RJZ1DK+Ua8Wyyh0WXWb8uHbdbYMjY1ChUztyC3Xeo7jsFj/OXTi1EVUzw4zTZphmyaXCG1/HwM/UNsWsnfnUYeS1DRe2x3RtA24jpnXZV1vu2Mlns92ykm23e0OQaK8XC6kTfBGBKci9t/UiAvHC9i/U3KwNV9bcHIqalqKxjewtk9OQ2Th8CuTUuTxLd68JEa+s3NvTz905PYmmfWnndLdOxNw5HZiMc9ppYYDlirFtb77BmESbDSQ33oYjWcrIthJVVyx2/IFNFWOuAdJ73lb51jXelfJ/NdvXI5K9i6revLveQuAFkecW9x6I9xews7DJwBNpsuBMhKwGqE4jMin7vuhVOW31WNR1mYlxkqBCiBcOGfRNyCCqeFuzGFdh1YhCIw+9iQuGKhp1Do8hjcOQIFN22q6GdANqd5kyxiXYY25akGWo1h1H0WfKeB19Furb9nLb7fVyC1eUj5gIk7hdvayKjoQ6ev3OmUyZ1u3ju5CSI+U29aH2SCnbSZQW4OsCBLPPbuvtjqlsz8JA8mVmjXuAB5INSPtiKr/rImtzYByjRme5kpJdVMrbd3E77YUQiorPEV82yHRQKj+8f2Jd/JM9az3BPxjjiHTuWcfTJFLtribJdvxKKnX5CnU2wahu+6YfvoeZbdXaX/lbTCY+bn53ZMqIoZZo0WIKvz0yZZ2vW5PpSFr7WYJpWxOMq5tJ7v5sJhg+DBF5OWs+HPvGk6fkn6/pn/4P</diagram></mxfile>
```
Comment (BjarkeMeier) [inline docs/images/PersistingChargesRuleSet_ProcessFlow.drawio:1]: Wauw - charges might be a CRUD domain but that's apparently not the same thing as being _simple_ :-D

**Review by BjarkeMeier — COMMENTED** [review]
This looks like a great documentation of the algorithm. I'm however not well enough to the stuff to approve. So I'll just leave my nit comments. Perhaps this could be presented to the entire core team together with @bemwen's "bar chart scenarios"?

File: docs/images/PersistingChargesRuleSet_ProcessFlow.drawio (hunk 2 above)
Comment (prtandrup) [reply docs/images/PersistingChargesRuleSet_ProcessFlow.drawio:1]: I assume you caught that the drawio file had more than one flow in it. I will upload a new version with only one flow.

File: docs/process-flows/README.md (hunk 1 above)
Comment (prtandrup) [reply docs/process-flows/README.md:8]: I've added a reference on /README, thx for input.

**Review by Mech0z — APPROVED** [review]

**Review by lasrinnil — APPROVED** [review]
LGTM! Perhaps we should update our architecture drawing in the near future? Ill bring this topic up on our next team dev session.

**Pull request merged**